app/Console/Commands/AutoSoftDeleteDisabledUsers.php

```diff
@@ -61,108 +64,83 @@ public function handle(): void
-                foreach (PrivateMessage::where('sender_id', '=', $user->id)->get() as $sent) {
-                    $sent->sender_id = 1;
-                    $sent->save();
-                }
-
-                // Removes UserID from PM if any and replaces with System UserID (1)
-                foreach (PrivateMessage::where('receiver_id', '=', $user->id)->get() as $received) {
-                    $received->receiver_id = 1;
-                    $received->save();
-                }
-
-                // Removes all Posts made by User from the shoutbox
-                foreach (Message::where('user_id', '=', $user->id)->get() as $shout) {
-                    $shout->delete();
-                }
-
-                // Removes all likes for user
-                foreach (Like::where('user_id', '=', $user->id)->get() as $like) {
-                    $like->delete();
-                }
-
-                // Removes all thanks for user
-                foreach (Thank::where('user_id', '=', $user->id)->get() as $thank) {
-                    $thank->delete();
-                }
+                $user->update([
+                    'can_upload'   => false,
+                    'can_download' => false,
+                    'can_comment'  => false,
+                    'can_invite'   => false,
+                    'can_request'  => false,
+                    'can_chat'     => false,
+                    'group_id'     => UserGroups::PRUNED->value,
+                    'deleted_by'   => User::SYSTEM_USER_ID,
+                ]);
+
+                Torrent::withoutGlobalScope(ApprovedScope::class)->where('user_id', '=', $user->id)->update([
+                    'user_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                Comment::where('user_id', '=', $user->id)->update([
+                    'user_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                Post::where('user_id', '=', $user->id)->update([
+                    'user_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                Topic::where('first_post_user_id', '=', $user->id)->update([
+                    'first_post_user_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                Topic::where('last_post_user_id', '=', $user->id)->update([
+                    'last_post_user_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                PrivateMessage::where('sender_id', '=', $user->id)->update([
+                    'sender_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                PrivateMessage::where('receiver_id', '=', $user->id)->update([
+                    'receiver_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                Invite::where('user_id', '=', $user->id)->update([
+                    'user_id' => User::SYSTEM_USER_ID,
+                ]);
+
+                Invite::where('accepted_by', '=', $user->id)->update([
+                    'accepted_by' => User::SYSTEM_USER_ID,
+                ]);
+
```

Comment:
```suggestion
```

These shouldn't be changed: https://github.com/HDInnovations/UNIT3D-Community-Edition/pull/3153